config/sync/webform.webform.contact.yml

```diff
@@ -208,7 +228,7 @@ handlers:
     label: 'Email confirmation'
     notes: ''
     handler_id: email_confirmation
-    status: true
+    status: false
```

Comment:
Do these need to be disabled if the webform is closed as a whole, or the environment they're running on isn't connected to an operating MTA service? Could be that they're to be all switched on a little later down the line but wanted to check

Reply:
I disabled this initially and then decided to close the contact form as a whole.  (The form is provided with the Webform module - but we do not use it.  It is in fact a sec vuln because it can act as an open relay)